src/code_actions.zig

```diff
@@ -70,6 +71,28 @@ pub const Builder = struct {
     }
 };
 
+fn handleNonCamelcaseFunction(builder: *Builder, actions: *std.ArrayListUnmanaged(types.CodeAction), loc: offsets.Loc) !void {
+    const identifier_name = offsets.locToSlice(builder.text(), loc);
+
+    const tree = builder.handle.tree;
+
+    const decl = (try analysis.lookupSymbolGlobal(builder.document_store, builder.arena, builder.handle, identifier_name, loc.start)) orelse return;
+
+    const name_token_idx = decl.nameToken();
+    const name_loc = offsets.tokenToLoc(tree, name_token_idx);
+
+    const new_text = try createCamelcaseText(builder.arena.allocator(), identifier_name);
+
+    const action1 = types.CodeAction{
+        .title = "make function name camelCase",
+        .kind = .SourceFixAll,
```

Comment:
I am unsure if this should be part of autofix (indicated by `SourceFixAll`). Can you set it to `QuickFix` while i think about it being included in autofix.